packages/miniflare/src/plugins/kv/index.ts

```diff
@@ -71,18 +80,29 @@ export const KV_PLUGIN: Plugin<
 			bindings.push(...(await getSitesBindings(options)));
 		}
 
+		if (isWorkersWithAssets(options)) {
```

Comment:
decided to keep the `assets` and `sites` code paths separate so we don't have to worry on the long term about breaking the sites behaviour